datafusion/expr/src/logical_plan/plan.rs

```diff
@@ -877,19 +877,20 @@ impl LogicalPlan {
                     input: Arc::new(inputs[0].clone()),
                 }))
             }
-            LogicalPlan::Explain(_) => {
+            LogicalPlan::Explain(e) => {
```

Comment:
I find we can correct transform `LogicalPlan::Analyze` in below
https://github.com/apache/arrow-datafusion/blob/340ecfdfe0e6667d2c9f528a60d8ee7fa5c34805/datafusion/expr/src/logical_plan/plan.rs#L871-L878
so I used the same approach in `LogicalPlan::Explain`